documentation/documentation/events/projections/custom.md

```diff
@@ -35,13 +35,13 @@ It comes of the way how Marten handles projection mechanism:
 <ol>
 <li>Try to find document that has the same <b>Id</b> as the value of the property selected from event (so eg. for <b>UserCreated</b> event it will be <b>UserId</b>).</li>
 <li>
-    If such document exists, then new record needs to be created. Marten by default tries to use <b>default constructor</b>. <br />
-    Default constructor doesn't have to be public, might be also private or protected. <br />
-    If class does not have the default constructor then it creates an uninitialized object (see <a href="https://docs.microsoft.com/en-us/dotnet/api/system.runtime.serialization.formatterservices.getuninitializedobject?view=netframework-4.8" target="_parent">more</a>).<br />
+    If no such document exists, then a new record needs to be created. Marten by default tries to use the <b>default constructor</b>. <br />
+    The default constructor doesn't have to be public, it can also private or protected. <br />
```

Comment:
Minor correction, change "it can also private or protected." to "it can also be private or protected."